src/config/express.js

```diff
@@ -86,6 +86,14 @@ app.use((req, res, next) => {
     return next(err);
 });
 
+// Log errors.
+if (config.env !== 'test') {
+    app.use((err, req, res, next) => {
+        winstonInstance.warn(err);
```

Comment:
FFR: Using `warn` because this will include both operational and programmer errors. In the future, may inspect the `err` and log to either `warn` or `error` accordingly.